src/components/Form/DataList/index.js

```diff
@@ -60,7 +60,6 @@ const DataList = ({
         aria-disabled={disabled}
         list={`autocomplete-list-${name}`}
         autoComplete="off"
-        onChange={(e) => onChange && onChange(e)}
```

Comment:
Defined already [here](https://github.com/LBHackney-IT/repairs-hub-frontend/pull/619/commits/4f76ee5bfeb12590939811d4203580c27aa1c33d#diff-638affd5558f723f78a3a8b508dda4df566d7e6abb45c4b54ffbc105ddd82225R58)